apps/settings/test/unit/app_starter_test.js

```diff
@@ -83,7 +78,7 @@ suite('AppStarter', function() {
       }).then(done, done);
     });
 
-    test('should use RootPanelHandler if the initial panel id is root',
+    test('should pass null customPanelHandler if the initial panel id is root',
```

Comment:
Same here if we don't use that argument, and can remove that logic, lets do that and this test would be unnecessary